cookbook/core/remote_flyte/customizing_resources.py

```diff
@@ -26,7 +25,11 @@
 # #. ``mem``
 # #. ``gpu``
 #
-# The ``storage`` resources option is not yet supported, but coming soon
+# It is not necessary to adjust admission controllers for gpu nodes, but they will need to be configured with a taint. 
```

Comment:
can we add that it should match the resource toleration specified in the config below?

Reply:
Done.